content/_data/changelogs/weekly.yml

```diff
@@ -13969,6 +13969,124 @@
   # pull: 6189 (PR title: Confirm that UMOMNEE2 is translated to UMOMNEE in legacy API)
   # pull: 6196 (PR title: Fix UtilTest#createDirectoriesInRoot() on mac)
 
+  - version: '2.333'
+    date: 2022-01-31
+    changes:
+      - type: TODO
+        category: developer
+        pull: 6231
+        authors:
+          - dependabot[bot]
+        pr_title: Bump XStream from 1.4.18 to 1.4.19
+        message: |-
+          Jenkins has upgraded the [XStream](https://x-stream.github.io/) library from 1.4.18 (released on August 22, 2021) to 1.4.19 (released on January 29, 2022). This maintenance release of XStream addresses the security vulnerability [CVE-2021-43859](https://x-stream.github.io/CVE-2021-43859.html), when unmarshalling highly recursive collections or maps causing a Denial of Service.
```

Comment:
lose comma after the link to the security vulnerability.
"...or maps that can cause a Denial of Service attack."

Reply:
Thanks!  I think it has already been fixed in 27a49a5c8e